File: NOVEMBER-2025/09-11-2025/p2169.js
var countOperations = function (num1, num2) {
    let res = 0;
    while (num1 && num2) {
        res += Math.floor(num1 / num2);
        num1 %= num2;
        [num1, num2] = [num2, num1];
    }
    return res;
};

File: NOVEMBER-2025/09-11-2025/p2169.rb
# @param {Integer} num1
# @param {Integer} num2
# @return {Integer}
def count_operations(num1, num2)
    count = 0

    while num1 > 0 && num2 > 0
        if num1 > num2
            cnt = num1 / num2
            num1 -= num2 * cnt
        else
            cnt = num2 / num1
            num2 -= num1 * cnt
        end

        count += cnt
    end

    count
end